backend/de.metas.acct.base/src/main/java-legacy/org/compiere/acct/DocLine_Inventory.java

```diff
@@ -100,6 +103,7 @@ public CostAmount getCreateCosts(final AcctSchema as)
 							.documentRef(CostingDocumentRef.ofInventoryLineId(get_ID()))
 							.qty(getQty())
 							.amt(CostAmount.of(this.costPrice, as.getCurrencyId()))
+							.isExplicitCostPrice(isExplicitCostPrice)
```

Comment:
.amt(CostAmount.zero(as.getCurrencyId())
.explicitCostPrice(explicitCostPriceBD != null ? CostAmount.of(explicitCostPriceBD, as.getCurrencyId() : null)